README.md

```diff
@@ -18,6 +18,7 @@ A pure Lua Neovim plugin for automatically rendering PlantUML diagrams in a loca
     auto_start = true,
     http_port = 8764,
     plantuml_server_url = "http://www.plantuml.com/plantuml",
+    auto_launch_browser = "once",
```

Comment:
The default should be "never", so fix this here, and also fix line 32 to show a different value.